ui/src/views/mint/github-step/steps/use-github.ts

```diff
@@ -0,0 +1,69 @@
+import { DropdownItem } from '@/components';
+import { useGithubStore } from '@/store';
+import { Octokit } from 'octokit';
+
+export const useGithub = () => {

```

Comment:
I recommend move this hook to the async thunk file and change it from hook to basic functions, you are able to retrieve the token directly in redux state and this would remove the circular dependency.